.gitlab-ci.yml

```
@@ -216,7 +234,10 @@ build_kernel_version_testing_arm64:
 build_circleci_runner:
   extends: [.x64]
   stage: build
-  except: [tags]
+  rules:
+    - if: $CI_COMMIT_TAG != null
+      when: never
+    - when: always
```

nit: this should be equivalent
```suggestion
  rules:
    - if: $CI_COMMIT_TAG == null
```

---

Indeed! Fixed